### ISSUE TITLE
`continue loop` gets minified to `continueloop`

### ISSUE THREAD

**Issue by gsingh93** [erikd/hjsmin#20]
I'm running into an issue with Yesod, which says it uses this library for minification. The `continue loop` in the following function gets minimized to `continueloop`.
```       
function filterListeners(inputs, listeners) {
    loop:
    for (var i = listeners.length; i--; ) {
        var listener = listeners[i];
        for (var j = inputs.length; j--; ) {
            if (listener.relevantInputs.indexOf(inputs[j].id) >= 0) {
                continue loop;
            }
        }
        listener.domNode.removeEventListener(listener.eventName, listener.func);
    }
}
```

**Comment by erikd** [comment]
Hi, I think I've fixed this in 4da88f7499aae39b6319e9104508857a7eb00134. Please test and verify.

**Comment by erikd** [comment]
Assuming this is fixed.
